<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件名处理器_YSP_V1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dir_select_Button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>83</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>路径选择</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>271</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>修改后结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="result_data_Text"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>461</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="init_dir_label">
       <property name="text">
        <string>请输入文件路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="init_dir_entry">
       <property name="text">
        <string>点击目录选择重命名文件目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="init_data_label">
       <property name="text">
        <string>请输入前缀：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="init_prefix_Text"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>请输入后缀：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="init_suffix_Text"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>279</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="modify_Button">
       <property name="text">
        <string>加前后缀</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resume_Button">
       <property name="text">
        <string>删前后缀</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rename_Button">
       <property name="text">
        <string>重新命名</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>451</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>日志</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="log_data_Text"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
